charts/dapr/charts/dapr_operator/templates/dapr_operator_service.yaml

```diff
@@ -52,19 +52,10 @@ metadata:
     {{- range $key, $value := .Values.global.k8sLabels }}
     {{ $key }}: {{ tpl $value $ }}
     {{- end }}
-{{- if or .Values.apiService.annotations .Values.global.prometheus.enabled }}
+{{- if .Values.webhookService.annotations }}
   annotations:
-    {{- if .Values.global.prometheus.enabled }}
-    prometheus.io/scrape: "true"
-    prometheus.io/port: "{{ .Values.global.prometheus.port }}"
-    prometheus.io/path: "/"
-    {{- end }}
-  {{- if .Values.webhookService.annotations }}
-    {{- range $key, $value := .Values.webhookService.annotations }}
-      {{ $key }}: "{{ $value }}"
-    {{- end }}
-  {{- end }}
-  {{- end }}
+{{ toYaml .Values.webhookService.annotations | indent 4}}
+{{- end }}
```

Comment:
how did this break the helm chart? How were you testing? Bc this revert changes it back to something that is broken tbh bc there should be a dash before the toYaml in the yaml manifest. @artursouza

Reply:
and this should be the same setup for all of the other services so how is this one broken and not the others?